media/redesign/stylus/components/userflags.styl

```diff
@@ -0,0 +1,8 @@
+/*
+    Some flags that adorn users, generally for display to site managers
+*/
+
+.banned {
+    color: #f00;
+    set-smaller-font-size();
+}
```

Comment:
I'm trying to move us towards having our stylus files based on individual components, could we name this file banned.styl after the component? Other user flags will get their own files. (Yes it feels weird to have a stylus file that has only one class declaration in it).